### PR TITLE
Add XYZ support to lmu2png, and improve layering accuracy

### PR DESCRIPTION
**Pull request by 20kdc**
Firstly, adds .xyz support to lmu2png.
Secondly, improves layering by reading & using the chipset flags if available, along with checking event layers, and such.

(TODO: Need to generate comparison images, or alternatively find someplace in the TestGame that deals with event layering.)

### CODE REVIEW

**Comment by carstene1ns** [comment]
Jenkins: ok to test.

File: lmu2png/src/main.cpp [hunk 1]
```diff
@@ -169,92 +276,50 @@ int main(int argc, char** argv) {
...
 			std::cout << "Chipset " << chipset_base << " can't be found." << std::endl;
 			exit(EXIT_FAILURE);
 		}
+		// Load flags.
+		memset(csflag + 0, cs.passable_data_lower[0], 1000);
+		memset(csflag + 1000, cs.passable_data_lower[1], 1000);
+		memset(csflag + 2000, cs.passable_data_lower[2], 1000);
+		memset(csflag + 3000, cs.passable_data_lower[3], 50);
+		memset(csflag + 3050, cs.passable_data_lower[4], 50);
+		memset(csflag + 3100, cs.passable_data_lower[5], 900);
```
Comment (Ghabry) [inline lmu2png/src/main.cpp]: Could you make this memset-stuff a bit less "magic number" heavy?

**Comment by Ghabry** [comment]
tbh I'm not a big fan of having another unique xyz reader. But is my fault for not finishing libxyzimage...

**Comment by Ghabry** [comment]
Next step: lmu thumbnailer \*g\*

**Pull request merged**